libvga dosen't exist on void. and it makes cross fail.

checking for /usr/lib/libvga.a... configure: error: cannot check for file existence when cross compiling

--- a/configure.ac	2020-10-22 20:19:24.633514760 +1100
+++ b/configure.ac	2020-10-22 20:19:46.917655818 +1100
@@ -126,10 +126,7 @@
 
 AS_IF([test "x$enable_mupdisp" != xno],
 	[
-	# Check if libvga.a is available for mupdisp
-	AC_CHECK_FILE([/usr/lib/libvga.a],[use_vga=yes],)
-	AC_CHECK_FILE([/usr/lib64/libvga.a],[use_vga=yes],)
-	test "$use_vga" != "yes" && AC_SUBST([LIBVGA],[-DNO_VGA_LIB])
+	AC_SUBST([LIBVGA],[-DNO_VGA_LIB])
 	]
 )
 
